manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ProcessingErrorResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.redhat.service.smartevents.manager.api.models.responses;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class ProcessingErrorResponse {
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ")
+    @JsonProperty("recorded_at")
+    private ZonedDateTime recordedAt;
+
+    @JsonProperty("headers")
```

Comment:
@kostola I'm not against it, but any particular benefit you see to separating headers and payload into two columns?

Reply:
Honestly just easier debugging when I look at the database, and clear separation between the two sets of information that come from the errors. I'd like to keep it this way.